### ISSUE TITLE
gracefully handle a request to pull back the issue associated with a  PR where there is no corresponding issue

### ISSUE THREAD

**Issue by mikeaintworkin** [thisaintwork/iqss_gh_reporting#42]
Verify that the we get back something consistent.
We should get back and empty list.

**Comment by mikeaintworkin** [comment]
```
-----
kickoff the Test



================================
--------------------------------
test: PR exists but there are no associated issues
External Exporters bug fix #9632
--------------------------------
Running /home/barry/PycharmProjects/iqss_gh_reporting/scripts/scaffolding/framework_02.py as the main program
Query dictionary
----
{
"query_str": "\n    query ($loginOrg: String!, $repo: String!, $firstFew: Int!, $startWith: String, $number: Int!) {\n        repository(followRenames:false, owner: $loginOrg, name: $repo) {\n                pullRequest (number: $number) {\n                    closingIssuesReferences(first: $firstFew, after: $startWith) {\n                        totalCount\n                        pageInfo {\n                            hasNextPage\n                            hasPreviousPage\n                            endCursor\n                    startCursor\n                        }\n                        nodes {\n                            ...issueFields\n                        }\n\n                    }\n                }\n        }\n    }\n        \n    fragment issueFields on Issue {\n        repository {\n            name\n        }\n        title\n        number\n        id\n        url\n        closed\n        closedAt\n        labels (first: 100) {\n          totalCount\n            nodes {\n                name\n            }\n        }\n    }\n    ",
"has_next_page_path": [
"repository",
"pullRequest",
"closingIssuesReferences",
"pageInfo",
"hasNextPage"
],
"start_with_path": [
"repository",
"pullRequest",
"closingIssuesReferences",
"pageInfo",
"endCursor"
],
"query_vars": {
"loginOrg": "IQSS",
"firstFew": 100,
"repo": "dataverse",
"number": 9632
}
}
----

results:
----
{
"repository": {
"pullRequest": {
"closingIssuesReferences": {
"totalCount": 0,
"pageInfo": {
"hasNextPage": false,
"hasPreviousPage": false,
"endCursor": null,
"startCursor": null
},
"nodes": []
}
}
}
}
----

Saving result to file:
/home/barry/Documents/github_reporting/framework_02/out/output.json
```

**Comment by mikeaintworkin** [comment]
we do get back and empty "nodes" list.
